tools/65-libwacom.rules.in

```diff
@@ -7,6 +7,9 @@ KERNEL!="event[0-9]*", GOTO="libwacom_end"
 ATTRS{name}=="* Consumer Control", GOTO="libwacom_end"
 ATTRS{name}=="* System Control", GOTO="libwacom_end"
 
+# HUION keyboard is not a tablet
+ATTRS{name}=="HUION * Keyboard", GOTO="libwacom_end"
```

Comment:
i wonder if we can just skip the `HUION` bit here? I don't recall a tablet/pad with the `Keyboard` prefix

Reply:
The two rightmost buttons on the touch bar send respectively TAB and SUPER+D keyboard events (wasn't able to reprogram those). Without this udev rules they get disabled

Reply:
Oh, a misunderstanding here - what I meant was this:
```suggestion
ATTRS{name}=="* Keyboard", GOTO="libwacom_end"
```
Because that should work just as well for the HUION device and I don't think we want any of the `Keyboard` devices tagged as pad anyway.